src/lustre/lustreAstDependencies.mli

```diff
@@ -64,6 +64,8 @@ val error_message: error_kind -> string
 
 type node_summary = ((int list) IntMap.t) IMap.t
 
+val pp_print_node_summary: Format.formatter -> int list IntMap.t IMap.t -> unit
+
```

Comment:
`pp_print_node_summary` is not used outside this module. Remove it from the interface file.

Reply:
addressed